css/appointment.css

```diff
@@ -0,0 +1,335 @@
+*{
+  box-sizing: border-box;
+}  
+
+.row::after{
+  content: "";
+  clear: both;
+  display: table;
+}
+
+[class*="col-"]{
+  float: left;
+  padding: 10px;
+}  
+
+[class*="col-"]{
+  width: 100%;
+}
+/* Medium devices (landscape tablets, 768px and up)*/
+@media only screen and (min-width: 768px){
+
+
+  .col-1 {
+    width: 8.33%;
+  }  
+  
+  .col-2 {
+    width: 16.66%;
+  }
+  
+  .col-3 {
+    width: 25%;
+  }
+  .col-4 {
+    width: 33.33%;
+  }
+  
+  .col-5 {
+    width: 41.66%;
+  }
+  
+  .col-6 {
+    width: 50%;
+  }
+  
+  .col-7 {
+    width: 58.33%;
+  }
+  
+  .col-8 {
+    width: 66.66%;
+  }
+  
+  .col-9 {
+    width: 75%;
+  }
+  
+  .col-10 {
+    width: 83.33%;
+  }
+  
+  .col-11 {
+    width: 91.66%;
+  }
+  
+  .col-12 {
+    width: 100%;
+  }
+  }
+  
+  /* Small devices (portrait tablets and large phones, 600px and up)*/
```

Comment:
Try putting a condition here for when e.g (min-width:600px and max-width : 768px)to ensure that you are not overlapping breakpoints. Research this to see the best approach.